core/qthelper.cpp

```diff
@@ -1143,6 +1143,11 @@ QString localFilePath(const QString &originalFilename)
 	return localFilenameOf.value(originalFilename, originalFilename);
 }
 
+// the water types need to match the watertypes enum above
+const QStringList waterTypes = {
+	gettextFromC::tr("Fresh"), gettextFromC::tr("Salty"), gettextFromC::tr("EN13319"), gettextFromC::tr("Salt"), gettextFromC::tr("use dc")
```

Comment:
should `use dc` be `Use DC`?

Reply:
``use dc`` is what was in the original QStringList. I agree though, `Use DC` would be more attractive - want me to update that?

Reply:
Done

Reply:
reading online: there is fresh water < EN13319 < salt water
why do we need `Salty`?

also should we order them by salinity level in this list?

Reply:
PTAL at the second comment.

Reply:
never mind, looks like "salty" was there before.